.github/workflows/gotest.yml

```diff
@@ -13,6 +13,7 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 2
+      - run: pip install git-changelog
       - run: |
           make check-commits
```

Comment:
Nitpick: to speed this up you could utilize native GHA action which will cache the Python modules for our job to make it super fast.